### DISCUSSION TITLE
電算中心系統開發維護工讀生招募中

### DISCUSSION THREAD

**Discussion by gnsJhenJie** [1101-GS4719-C/Python-Community-GS4719-C#665] [閒聊八卦]
大家如果對開發與維護網頁有興趣又想賺點零用錢的話可以來嘗試看看,  雖然大部分系統是以PHP開發, 不過也有一個聊天機器人是用Python的Flask開發的~ 

電算中心系統開發維護工讀生招募中！
👉工作內容
系統開發，工作內容除了要維護客服系統網站，也會有新網站與系統的開發
👉徵選條件
熟悉 html/ PHP / javascript 或其他網頁程式語言
有 Laravel 或 Linux 經驗尤佳
有其他類似的網管工讀經驗尤佳
若完全沒有經驗，有心學習者可議
積極、要能自主學習
👉徵選流程
培訓期間為暑假期間約2個月，不另支薪
通過培訓且表現優良者始得錄取
👉額外福利
每週開會時有便當٩(˃̶͈௰˂̶͈)و
👉報名方式
採Email報名
郵件標題：應徵「電算中心系統開發維護工讀生」
郵件內容需附上：
1.姓名/系級
2.列出你會的程式語言或相關技術，並以5分為上限評量各語言熟悉的程度
（例如：php 3分、html 5分、Vue.js 2分、其他網頁程式語言4分、linux 4分...）
3.會不會ps / 以拉拉等電腦繪圖軟體（不會也沒關係）
4.網站開發相關經驗或有網頁作品也可以附上
5.111年上學期是否有其他校內聘雇
並將信件寄到[ncuccstusd@gmail.com](mailto:ncuccstusd@gmail.com)信箱
👉email報名截止時間 6/17
面試及開會日期時間再另行通知唷_(:з」∠)_

**Comment by treeann** [comment]
電算中心的工讀生職缺感覺很吸引人ㄝ!

**Comment by minniewei** [comment]
感覺真的超吸引人的！！但就是培訓有點久...

**Comment by jcpan821** [comment]
可惜我還不夠強

**Comment by alexthebigpp** [comment]
感覺可以學到很多東西，可惜要大4了

**Comment by 110409538** [comment]
暑假該賺錢了吧!
